### ISSUE TITLE
Missing support for new break features of MLS 3.6

### ISSUE THREAD

**Issue by m-kessler** [ErykMroczek/mofmt#6]
Modelica Language Specificaton 3.6 introduced two new features involving the existing `break` keyword, which are not formatted correct by mofmt:

1. [Removing modifiers](https://specification.modelica.org/maint/3.6/inheritance-modification-and-redeclaration.html#removing-modifiers-break)
2. [Selective model extension](https://specification.modelica.org/maint/3.6/inheritance-modification-and-redeclaration.html#selective-model-extension)

Here is an example using those new features:
```
partial model ConditionalExtends
  extends Modelica.Blocks.Examples.PID_Controller(
    break loadTorque,
    inertia1(J=break),
    break connect(speedSensor.w, PI.u_m));
end ConditionalExtends;
```

Running it through mofmt 0.3.1 gives no errors, but the output is not satisfying:

```
partial model ConditionalExtends

  extends Modelica.Blocks.Examples.PID_Controller(break loadTorque,
    inertia1(J = break), break connect(speedSensor.w, PI.u_m));

end ConditionalExtends;
```

**Comment by ErykMroczek** [comment]
This and other reported issues will be fixed in `0.3.2`.

**Comment by m-kessler** [comment]
Thanks for the quick fixes. v0.3.2 works like a charm.

By the way, mofmt is a great tool and very helpful. Nicely done!
